aries-site/src/pages/components/card.mdx

```diff
@@ -83,20 +83,6 @@ Learn more about [Dashboard Templates](/templates/dashboards).
   alt="HPE Design System Cards Dashboard Example"
 />
 
-### Navigation Example
-
-A grid of cards can provide a simple set of easy to navigate items. In this example,
-the cards are a uniform, homogenous collection of services, which can be browsed
-and easily digested. A click on card leads to that service's view and allow the user
-to explore further.
-
-<ExampleImagePreview
-  label="See Cards as Navigation"
-  href="/templates/hub-and-spoke-navigation"
```

Comment:
It's a bummer to see this go. It feels like worthwhile content and placement. However, we've blown up the previous approach to representing navigation as being use case driven. I don't know that leaving the content and directing to '/templates/drill-down-navigation#card-example' helps at all. 

I'm okay with deleting for now given the current approach to navigation, but feel like we abandoned something that was somewhat delivering what consumers were desiring "lead with use cases and give the pattern a name."